src/plugins/stake-plugins/uniswapV2/policies/VelodromeV2StakePolicy.ts

```diff
@@ -0,0 +1,728 @@
+              })
+              cacheTxMetadata(result.hash, tokenBCurrencyCode, {
+                name: metadataName,
+                category: 'Transfer:Staking',
+                notes: 'Remove liquidity from ' + metadataLpName + ' - LP'
+              })
+            })(gasLimitAcc('500000'))
+        )
+      }
+
+      //
+      // Claim
+      //
+
+      if (action === 'claim') {
+        // 1. Claim rewards
+        txs.build(
+          (gasLimit =>
+            async function claimReward({ signer }) {
+              const result = await stakingContract.connect(signer).getReward(signer.address, {
+                gasLimit,
+                gasPrice,
+                nonce: nextNonce()
+              })
+
+              policyInfo.rewardAssets
+                .map(asset => asset.currencyCode)
+                .forEach(currencyCode => {
+                  cacheTxMetadata(result.hash, currencyCode, {
+                    name: metadataName,
+                    category: 'Income:Staking',
+                    notes: `Claimed rewards from ${metadataLpName}`
+                  })
+                })
+              cacheTxMetadata(result.hash, parentCurrencyCode, {
+                name: metadataName,
+                category: 'Expense:Fees',
+                notes: `Claimed rewards from ${metadataLpName}`
+              })
+            })(gasLimitAcc('240000'))
+        )
+      }
+
+      //
+      // Calculate the fees
+      //
+
+      // Calculate the networkFee as the gasLimit * gasPrice in the native token
+      const networkFee = gasLimitAcc().mul(gasPrice).toString()
+      allocations.push({
+        allocationType: 'networkFee',
+        pluginId: policyInfo.parentPluginId,
+        currencyCode: policyInfo.parentCurrencyCode,
+        nativeAmount: networkFee
+      })
+
+      //
+      // Return the quote
+      //
+
+      // Construct an approve function which executes transaction workflow (txBuilder)
+      const approve: () => Promise<void> = async () => {
+        await txs.run()
+      }
+
+      return {
+        allocations,
+        approve
+      }
+    },
+    async fetchStakePosition(request: StakePositionRequest): Promise<StakePosition> {
+      const { stakePolicyId, wallet, account } = request
+      const signerSeed = await account.getDisplayPrivateKey(wallet.id)
+
+      const policyInfo = pluginInfo.policyInfo.find(p => p.stakePolicyId === stakePolicyId)
+      if (policyInfo == null) throw new Error(`Stake policy '${stakePolicyId}' not found`)
+
+      const tokenACurrencyCode = policyInfo.stakeAssets[0].currencyCode
+      const tokenBCurrencyCode = policyInfo.stakeAssets[1].currencyCode
+      const isTokenANative = tokenACurrencyCode === policyInfo.parentCurrencyCode
+      const isTokenBNative = tokenBCurrencyCode === policyInfo.parentCurrencyCode
+
+      // Get the signer for the wallet
+      const signerAddress = await eco.makeSigner(signerSeed).getAddress()
```

Comment:
This `await` is missing in `cemeteryPolicy`. Does `cemeteryPolicy` need to be fixed?